backend/gsr_booking/admin.py

```diff
@@ -3,6 +3,29 @@
 from gsr_booking.models import GSR, Group, GroupMembership, GSRBooking, Reservation
 
 
+class GroupMembershipInline(admin.TabularInline):
+    model = GroupMembership
+    extra = 0
+
+    readonly_fields = ["name"]
+
+    def name(self, obj):
+        return obj.user.get_full_name()
```

Comment:
Probably fine but what if we made this more verbose for better searching/viewing? We only use Labs group at the moment but what if we displayed both username and group name for viewing and searching.

Reply:
This is inline for a group. AKA the group info will be at the top of the page when anyone is looking here